server/plugin/utils.go

```diff
@@ -208,32 +208,41 @@ func (p *Plugin) getConnectAccountFirstMessage() string {
 	return fmt.Sprintf(constants.ConnectAccountFirst, fmt.Sprintf(constants.ConnectAccount, p.GetPluginURLPath(), constants.PathOAuthConnect))
 }
 
-func (p *Plugin) ParseSubscriptionsToCommandResponse(subscriptionsList []*serializers.SubscriptionDetails, channelID string) string {
+func (p *Plugin) ParseSubscriptionsToCommandResponse(subscriptionsList []*serializers.SubscriptionDetails, channelID, createdBy, userID string) string {
 	var sb strings.Builder
 
 	if len(subscriptionsList) == 0 {
 		sb.WriteString(constants.NoSubscriptionFound)
 		return sb.String()
 	}
 
-	sb.WriteString("###### Board subscription(s) for this channel\n")
-	sb.WriteString("| Subscription ID | Organization | Project | Event Type |\n")
-	sb.WriteString("| :-------------- | :----------- | :------ | :--------- |\n")
+	sb.WriteString("###### Board's subscription(s)\n")
```

Comment:
```suggestion
	sb.WriteString("###### Boards subscription(s)\n")
```
I think there's no need of apostrophe anywhere where you've written `Board's`